recommender/static/html/recommender.html

```diff
@@ -116,11 +115,11 @@
                         <%include file="resourcebox.html" args="id=elem['id'],title=elem['title'],votes=elem['votes'],url=elem['url'],description=elem['description'],descriptionText=elem['descriptionText'],isProblematic=flag_mode,problematicReason=reason,voteMode=vote_mode,endorseMode=endorse_mode,endorsedReason=endorsed_reason" />
                     % endfor
                 </div>
-                <div class="recommender_resourceAddButton" role="button" tabindex="0">Add new resource &gt;&gt;</div>
-                <div class="recommender_resourceExportButton" role="button" tabindex="0">Download resources</div>
-                <div class="recommender_resourceImportButton" role="button" style='display: none;' tabindex="0" aria-hidden="true">Upload resources</div>
+                <div class="recommender_resourceAddButton" role="button" tabindex="0">${_("Add new resource")} &gt;&gt;</div>
```

Comment:
should the `&gt;` be a part of the `gettext` string? It seems having it hardcoded after the `gettext` could be problematic with RTL languages

Reply:
```suggestion
                <div class="recommender_resourceAddButton" role="button" tabindex="0">${_("Add new resource &gt;&gt;")}</div>
```

Yeah, it's safe to include it.